src/main/resources/mcmod.info

```diff
@@ -2,7 +2,7 @@
 {
   "modid": "minecolonies",
   "name": "MineColonies",
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you!",
+  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! This is a new typo: jsdoifjsdofijsofdij iosudhfishdfhusidf.isudhfishifdhsidfh, isduhfsiudfhisdhf isudhfiushdfihsdifu iuhsdifhsiufdhiusdfh. hdssdifhiudshfiusdiuhdsifdsih. iudsifhsifhihfdsiuhdsfhidsh.",
```

Comment:
### Infos:
- Typo: In word 'iudsifhsifhihfdsiuhdsfhidsh'
- Typo: In word 'NPC's'
- Typo: In word 'iosudhfishdfhusidf'
- Typo: In word 'iuhsdifhsiufdhiusdfh'
- Typo: In word 'isudhfiushdfihsdifu'
- Typo: In word 'isduhfsiudfhisdhf'
- Typo: In word 'Minecraft'
- Typo: In word 'hdssdifhiudshfiusdiuhdsifdsih'
- Typo: In word 'jsdoifjsdofijsofdij'
- Typo: In word 'townhall'
- Typo: In word 'isudhfishifdhsidfh'